demo/halcyon-api.json

```diff
@@ -0,0 +1,2532 @@
+{
```

Comment:
can change this to https://api.halcyon.blue/static/redoc# present here

Reply:
We won't be able to fetch this in as this will require authentication.